contracts/utils/CloneFactory.sol

```diff
@@ -33,21 +55,4 @@ contract CloneFactory {
             result := create(0, clone, 0x37)
         }
     }
-
-    function isClone(address target, address query) internal view returns (bool result) {
-        bytes20 targetBytes = bytes20(target);
```

Comment:
we are going to use this contract for future proxy deployments, inculding D2D, I don't know are we going to use this function in the future or not, but let's keep it just in case

Reply:
Okay, I'll write a small test to cover this.